python_src/src/lamp_py/performance_manager/pipeline.py

```diff
@@ -120,7 +89,15 @@ def start() -> None:
 
     # configure the environment
     os.environ["SERVICE_NAME"] = "performance_manager"
-    validate_environment()
+    validate_environment(
+        required_variables=[
+            "SPRINGBOARD_BUCKET",
+            "SERVICE_NAME",
+            "ALEMBIC_DB_NAME",
+        ],
+        optional_variables=["PUBLIC_ARCHIVE_BUCKET"],
```

Comment:
i added the public archive as an optional variable. it will be logged if it exists.